exercises/luhn-trait/src/lib.rs

```diff
@@ -1 +1,14 @@
+pub trait Luhn {
+    fn valid_luhn(&self) -> bool;
+}
 
+/// Here is the example of how to implement custom Luhn trait
+/// for the &str type. Naturally, you can implement this trait
+/// by hand for the every other type presented in the test suite,
+/// but you solution will fail if a new type is presented.
+/// Perhaps there exists a better solution for a problem?
```

Comment:
I think "a problem" should be "this problem" or "the problem", otherwise it might not refer to the problem presented in the previous sentence.